tests/pyccel/test_pyccel.py

```diff
@@ -297,6 +324,13 @@ def test_imports_compile(language):
 #------------------------------------------------------------------------------
 @pytest.mark.xdist_incompatible
 def test_imports_in_folder(language):
+    if language == 'python':
+        current_folder = os.path.abspath(__file__)
+        result = current_folder.split('/')
+        result = result[:-1]
+        current_folder = "/".join(result)
+        if os.path.exists(current_folder+"/scripts/py/folder1") is False:
+            shutil.copytree(current_folder+"/scripts/folder1", current_folder+"/scripts/py/folder1")
```

Comment:
Why do you need to copy the files?

Reply:
because the generated python code is in the folder `/scripts/py` and the import are in the `scripts` folder I copy them to make the code work

Reply:
Ah is this a special case for examples with multiple folders? Have you run a test without pytest to be sure that you are testing the generated code and not the copied code?

Reply:
i think this test not working well, I'm gonna try to fix it

Reply:
this one is fixed